WordPress/src/main/res/values/strings.xml

```diff
@@ -76,6 +76,12 @@
     <string name="learn_more">Learn more</string>
     <string name="posting_post">Posting \"%s\"</string>
 
+    <!-- MediaPicker -->
+    <string name="add_to_post">Add to Post</string>
+    <string name="media_picker_title">Select media</string>
+    <string name="take_photo">Take a photo</string>
+    <string name="capture_video">Capture a video</string>
```

Comment:
Maybe the photo/video strings should match. 'Take a photo' and 'Take a video' maybe?